pkg/cmd/server/crypto/extensions/serversigning.go

```diff
@@ -0,0 +1,42 @@
+package extensions
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+)
+
+var (
+	// OpenShiftServerSigningOID is the OpenShift assigned OID arc for certificates signed by the OpenShift server.
+	OpenShiftServerSigningOID = oid(OpenShiftOID, 100)
+	// OpenShiftServerSigningServiceOID describes the IANA arc for extensions to server certificates generated by the
+	// OpenShift service signing mechanism. All elements in this arc should only be used when signing server certificates
+	// for use under a service.
+	OpenShiftServerSigningServiceOID = oid(OpenShiftServerSigningOID, 2)
+	// OpenShiftServerSigningServiceUIDOID is an x509 extension that is applied to server certificates generated for services
+	// representing the UID of the service this certificate was generated for. This value is not guaranteed to match the
+	// current service UID if the certificates are in the process of being rotated out. The value MUST be an asn.1 encoded
```

Comment:
Was trying to imply that you are not guaranteed to have it be equal immediately.  And right now, the SSCC doesn't regenerate certs in all cases (deletion of the secret) so I wanted to add weasel words.